warp10/src/main/java/io/warp10/script/MemoryWarpScriptStack.java

```diff
@@ -616,15 +618,23 @@ public void exec(String line) throws WarpScriptException {
             }
             multiline.append(stmt);
             continue;
-          } else if (WarpScriptStack.COMMENT_END.equals(stmt)) {
+          } else if (!allowLooseBlockComments && WarpScriptStack.COMMENT_END.equals(stmt)) {
             if (!inComment.get()) {
               throw new WarpScriptException("Not inside a comment.");
             }
             inComment.set(false);
             continue;
+          } else if (allowLooseBlockComments && stmt.startsWith(WarpScriptStack.COMMENT_START) && stmt.endsWith(WarpScriptStack.COMMENT_END)) {
```

Comment:
Add some examples in a comment above each conditional so it is easy to understand what situation each one deals with.